uwblueprint/starter-code-v2
# Fix several minor issues

## Notion ticket link

N/A - fixing a couple of small issues that came up when testing the release branch
* Resolve `body-parser` deprecation warning message
* Fix casing issue in Python backend's openapi.json, `userId` -> `user_id`
* Allow empty list to be supplied as entity's `string_array_field` and simplify validation checks in Python backend
* Add newline to Logout.tsx (lint failure was causing frontend compile to fail)



## Implementation description
* N/A



## Steps to test
1. af26a93: Run the TypeScript backend, verify that this deprecation warning is not logged
```
body-parser deprecated undefined extended: provide extended option
```
2. 56575c9: Run the Python backend and navigate to `/api-docs`, try a GET user request with a `user_id` query parameter and a DELETE user request with a `user_id` query parameter
3. 286fa7b: Using the Python backend, try POST and PUT entities and users requests with missing or improperly typed fields and verify that 400 status codes are returned
4. 66a2797: Trivial (tested when creating the release already)
5. acf63c7: Renamed TS storageService.ts to fileStorageService.ts for consistency with Python backend



## What should reviewers focus on?
* Review commit by commit, each fixes a different issue


## Checklist
